04_environment/index.tex

```diff
@@ -1 +1,400 @@
-\chapter{Game Environment}
\ No newline at end of file
+\chapter{Environment}
+\section{Foraging}
+\subsection{Foraging Background}
+
+Foraging is a base requirement that is needed as it is a method of obtaining more resources from the starting amount.Foraging is based on the concept of inputting resources to get a return.\\
+
+\subsubsection{Underlying concept}
+The goal of foraging in this “game” is to introduce a certain dilemma to the agents to further understand the behaviours of multiple (6) intelligent agents working together to reach a goal with different personal goals. By having 2 methods of foraging it introduces the concept of decision making between the islands allowing for strategic interactions to maximise personal benefits whilst still reaching the overarching goal of survival.\\  
```

Comment:
why "game" ? I would say,  simply, game. (without quotation marks)

also do not write 6 but simply write six..

similarly,: **two methods of foraging**...

avoid using numbers unless you have a good reason to do so

Reply:
Following might sound a bit more academic, concise and formal - suggested rewrite:

The primary objective of foraging in this game is to introduce a dilemma to the agents to investigate their behaviour and whether they work together despite different personal goals.

Reply:
Suggested rewrite: 

Including two methods of foraging promotes decision making among the islands, allowing for strategic interactions to maximise personal benefits whilst still reaching the overarching goal of survival.